src/profiler/configuration.c

```diff
@@ -290,7 +290,7 @@ static void validate_op(MVMThreadContext *tc, validatorstate *state) {
                     break;
 
                 default:
-                    MVM_exception_throw_adhoc(tc, "STRUCT_SELECT string length %llu (index %d) NYI or something", string_length, string_idx);
+                    MVM_exception_throw_adhoc(tc, "STRUCT_SELECT string length %lu (index %d) NYI or something", string_length, string_idx);
```

Comment:
I think both `%llu` and `%lu` are wrong as their meaning is platform dependent. MVMuint64 is an alias for uint64_t and according to `man inttypes.h` the PRIu64 macro is the correct choice: `"STRUCT_SELECT string length " PRIu64 " (index %d) NYI or something"`